appveyor.yml

```diff
@@ -0,0 +1,16 @@
+---
+clone_depth: 10
+install:
+  - SET PATH=C:\Ruby%ruby_version%\bin;%PATH%
+  - appveyor DownloadFile http://dl.bintray.com/oneclick/OpenKnapsack/x64/openssl-1.0.1l-x64-windows.tar.lzma
```

Comment:
Why 1.0.1l? It is fairly outdated... (Official support for 1.0.1 branch will cease at the end of this year.)